test/CMakeLists.txt

```diff
@@ -1,29 +1,11 @@
 cmake_policy(SET CMP0015 NEW)
 
 aux_source_directory(. SRC_LIST)
+aux_source_directory(contracts SRC_LIST)
+aux_source_directory(libsolidity SRC_LIST)
 
 get_filename_component(TESTS_DIR "${CMAKE_CURRENT_SOURCE_DIR}" ABSOLUTE)
 
-macro (add_sources)
```

Comment:
What was this macro actually doing?    Just adding all subdirectories at some point in the past, but not used anymore?